<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>546</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>235</red>
        <green>241</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>175</red>
        <green>203</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>235</red>
        <green>241</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>175</red>
        <green>203</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>175</red>
        <green>203</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>175</red>
        <green>203</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Bot-assistant</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="grid">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Дополнительный вывод:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Чат с ботом:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QListWidget" name="chat">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="additional">
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>________________$$____________$$
_______________$___$________$___$
_______________$_____$$$$$$_____$
_______________$____sss___sss____$
_______________$____іі_____іі_____$
______________$_______$$$________$
__$$$$$$$$_____$_______$________$
$$________$_______$$_________$$
_$_________$_____$___$$$$$$___$
____$______$____$__$________$__$
____$_____$____$__$__________$__$
___$____$___$$$$__$__________$__$$$$
__$___$____$____$__$________$___$___$
__$__$_____$____$__$________$__$____$
_$___$______$____$__$____$_$__$____$
___$__$______$____$___$_$_____$___$
____$___$$$$$_$___$___$_$____$___$
_______$$$$$_$____$____$_____$____$
_____________$$$_$_____$______$_$$$
__________________$$$$___$$$$$</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="btn2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>10</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Выход</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
           </font>
          </property>
          <property name="text">
           <string>Загрузить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>10</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>Сохранить изменения</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Настройки</string>
    </property>
    <widget class="QMenu" name="choose_user">
     <property name="title">
      <string>Выбор Пользователя</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="choose_user"/>
    <addaction name="new_user"/>
    <addaction name="change_settings"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>О программе</string>
    </property>
    <addaction name="help"/>
    <addaction name="developers"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="new_user">
   <property name="text">
    <string>Новый пользователь...</string>
   </property>
  </action>
  <action name="change_settings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Настройки текущего пользователя</string>
   </property>
  </action>
  <action name="help">
   <property name="text">
    <string>Помощь</string>
   </property>
  </action>
  <action name="developers">
   <property name="text">
    <string>О разработчиках</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
